File: LeetCode/Sorting/Easy/minimum-number-of-moves-to-seat-everyone.js
/**
 * @param {number[]} seats
 * @param {number[]} students
 * @return {number}
 */
var minMovesToSeat = function(seats, students) {
    seats.sort((a,b)=>b-a);
    students.sort((a,b)=>b-a);
    let moves = 0;
    for(let i=0;i<=seats.length-1;i++)
      moves += Math.abs(seats[i]- students[i]);

    return moves;
};

File: LeetCode/Sorting/Easy/minimum-number-of-moves-to-seat-everyone.java

class Solution {
    public int minMovesToSeat(int[] seats, int[] students) {
        Arrays.sort(seats);
        Arrays.sort(students);
        int diff = 0;
        int sum =0;
        int n = seats.length;

        for(int i=0;i<=n-1;i++){
           diff = Math.abs(seats[i]- students[i]);
           sum = sum + diff;
        }

      return sum;
    }

}
